### PR TITLE
ode 0.16.3

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
@chenrui333 Any idea how we should handle this license?

File: Formula/ode.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 class Ode < Formula
   desc "Simulating articulated rigid body dynamics"
   homepage "https://www.ode.org/"
-  url "https://bitbucket.org/odedevs/ode/downloads/ode-0.16.2.tar.gz"
-  sha256 "b26aebdcb015e2d89720ef48e0cb2e8a3ca77915f89d853893e7cc861f810f22"
+  url "https://bitbucket.org/odedevs/ode/downloads/ode-0.16.3.tar.gz"
+  sha256 "c741dbf49bfc468ce2964824e5bc3f906ea956b1ae3594c50d351c383f03c413"
+  # Second license option: https://bitbucket.org/odedevs/ode/src/master/LICENSE-BSD.TXT
+  license any_of: ["LGPL-2.1-or-later", :cannot_represent]
```
Comment (chenrui333) [inline Formula/ode.rb:7]:
```suggestion
  license any_of: ["LGPL-2.1-or-later", "BSD-3-Clause"]
```

**Comment by Porkepix** [comment]
The file specifies this is a modified BSD 3-clause and third-party libraries use even other licences, so is it ok to specify only this?

**Comment by SMillerDev** [comment]
I'd say `:cannot_represent` but fedora says: https://src.fedoraproject.org/rpms/ode/blob/rawhide/f/ode.spec

**Comment by chenrui333** [comment]
build failure on macos intel chip

```
  /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/System/Library/Frameworks/Security.framework/Headers/cssmconfig.h:51:18: error: typedef redefinition with different types ('uint64_t' (aka 'unsigned long long') vs 'duint64' (aka 'unsigned long'))
  typedef uint64_t uint64;
                   ^
  ./typedefs.h:39:17: note: previous definition is here
  typedef duint64 uint64;
                  ^
  mv -f .deps/collision_convex_trimesh.Tpo .deps/collision_convex_trimesh.Plo
  mv -f .deps/collision_libccd.Tpo .deps/collision_libccd.Plo
  1 error generated.
```

**Pull request closed without merging**